### PR TITLE
Issue #SH-652 fix: Modified code for card click event inside view-all component

### PR DESCRIPTION
**Pull request by sudip-mukherjee**

### CODE REVIEW

File: src/app/client/src/app/modules/content-search/components/view-all/view-all.component.ts [hunk 1]
```diff
@@ -432,7 +432,12 @@ export class ViewAllComponent implements OnInit, OnDestroy, AfterViewInit {
       const enrolledCourseCount = _.get(courseData, 'enrolledCourses').length;
       this.noResult = false;
       this.totalCount = enrolledCourseCount;
-      this.searchList = this.formatSearchresults(_.orderBy(_.get(courseData, 'enrolledCourses'), ['enrolledDate'], ['desc']));
+      const sortedData = _.map(_.orderBy(_.get(courseData, 'enrolledCourses'), ['enrolledDate'], ['desc']), (val) => {
+        const value = _.get(val, 'content');
+        value['mimeType'] = 'application/vnd.ekstep.content-collection';
```
Comment (vinukumar-vs) [inline src/app/client/src/app/modules/content-search/components/view-all/view-all.component.ts:437]: This should not be hardcoded. We have to take it mimetype from val.mimetype if it available.

File: src/app/client/src/app/modules/content-search/components/view-all/view-all.component.spec.ts [hunk 2]
```diff
@@ -199,7 +199,11 @@ describe('ViewAllComponent', () => {
   it('should process the data if view-all is clicked from My-Courses section', () => {
     const courseService = TestBed.get(CoursesService);
     component.sectionName = 'My courses';
-    const sortedData = _.orderBy(_.get(Response, 'enrolledCourseData.enrolledCourses'), ['enrolledDate'], ['desc']);
+    const sortedData = _.map(_.orderBy(_.get(Response, 'enrolledCourseData.enrolledCourses'), ['enrolledDate'], ['desc']), (val) => {
```
Comment (vinukumar-vs) [inline src/app/client/src/app/modules/content-search/components/view-all/view-all.component.spec.ts:202]: Why it is different than component.ts code?

File: src/app/client/src/app/modules/content-search/components/view-all/view-all.component.ts (hunk 1 above)
Comment (sudip-mukherjee) [reply src/app/client/src/app/modules/content-search/components/view-all/view-all.component.ts]: Done.

File: src/app/client/src/app/modules/content-search/components/view-all/view-all.component.spec.ts (hunk 2 above)
Comment (sudip-mukherjee) [reply src/app/client/src/app/modules/content-search/components/view-all/view-all.component.spec.ts:202]: no sir. both logic of sorting is same.

**Pull request merged**